src/reflect/scala/reflect/internal/Types.scala

```diff
@@ -4549,7 +4549,12 @@ trait Types
 
   /** Members which can be imported into other scopes.
    */
-  def importableMembers(pre: Type): Scope = pre.members filter isImportable
+  def importableMembers(pre: Type): Scope = {
```

Comment:
I think this change is too risky for 2.11.9. How about moving it to the invocation of `importableMembers` in the repl?